test/functionalTest/harnessFunctions.sh

```diff
@@ -595,22 +625,200 @@ function dbInsertEntity()
 #   --host        <host>           (default: localhost)
 #   --port        <port>           (default: $CB_PORT)
 #   --url         <URL>            (default: empty string)
+#   --urlParams   <params>         (URI parameters 'in' URL-string)
 #   --payload     <payload>        (default: NO PAYLOAD. Possible values: [filename | "${string}"])
 #   --in          (input payload)  (default: xml => application/xml, If 'json': application/json)
 #   --out         (output payload  (default: xml => application/xml, If 'json': application/json)
 #   --json        (in/out JSON)    (if --in/out is used AFTER --json, it overrides) 
 #   --httpTenant  <tenant>         (tenant in HTTP header)
+#   --tenant      <tenant>         (tenant in HTTP header)
 #   --servicePath <path>           (Service Path in HTTP header)
-#   --urlParams   <params>         (URI parameters 'in' URL-string)
 #   --xauthToken  <token>          (X-Auth token value)
 #   --origin      <origin>         (Origin in HTTP header)
-#   --verbose                      (verbose output)
 #   --noPayloadCheck               (don't check the payload)
+#   --header      <HTTP header>    (more headers)
+#   -H            <HTTP header>    (more headers)
```

Comment:
Great! More flexibility...

(In fact, `--origin` could have been implemented using `-H`, but let's keep it as it is now).

NTC